.circleci/config.yml

```diff
@@ -124,6 +124,9 @@ jobs:
             - /root/.cache/pip
             - .mypy_cache
             - .pytest_cache
+      - run:
+          name: Install gpg (required by codecov orb)
+          command: apt update && apt install -y gpg
       - codecov/upload:
           when: always
           file: 'test-reports/coverage.xml'
```

Comment:
what's the report.xml created above used for then?

Reply:
It's used to generate the test summary page: https://app.circleci.com/pipelines/github/ESMValGroup/ESMValCore/9756/workflows/2169fe35-d00d-448d-826f-eeaa47109d51/jobs/42339/tests

That looks quite boring for this pr, but if there are some failing tests or gives you a nice quick overview without having to scroll through lots of terminal output.

Reply:
aha! Nice one then, thanks